cvat-core/src/organization.ts

```diff
@@ -234,24 +319,9 @@ Object.defineProperties(Organization.prototype.members, {
             checkObjectType('pageSize', pageSize, 'number');
 
             const result = await serverProxy.organizations.members(orgSlug, page, pageSize);
-            await Promise.all(
-                result.results.map((membership) => {
-                    const { invitation } = membership;
-                    membership.user = new User(membership.user);
-                    if (invitation) {
-                        return serverProxy.organizations
-                            .invitation(invitation)
-                            .then((invitationData) => {
-                                membership.invitation = invitationData;
-                            })
-                            .catch(() => {
-                                membership.invitation = null;
-                            });
-                    }
-
-                    return Promise.resolve();
-                }),
-            );
+            result.results = result.results.map((rawMembership: SerializedMembershipData) => new Membership(
+                rawMembership,
+            ));
```

Comment:
Instead of redefining ``result.results`` better to create a new identificator. 

const memberships = .... 
memberships.count = result.count;
return memberships;